docs/index.md

```diff
@@ -14,3 +14,12 @@ Teku:
 * Provides enterprise focused metrics with Prometheus.
 * Has REST APIs for managing Eth 2.0 node operations.
 * Has external key management to manage validator signing keys.
+
+## New to Ethereum 2.0?
+
+Get started by installing Teku or running Teku with Docker.
+You can:
+
+* [Install the binary distribution.](HowTo/Get-Started/Installation-Options/Install-Binaries.md)
```

Comment:
I would order from the easier to the harder: Use Docker, use binaries, build source code.